packages/legacy-workbench/src/js/angular/guides/guide-utils.js

```diff
@@ -2,6 +2,20 @@ import {select} from 'd3';
 
 const d3 = {select};
 const GuideUtils = (function () {
+
+    /**
+     * Checks whether a given DOM element is visible in the document.
+     *
+     * @param {HTMLElement|null} element - The DOM element to check.
+     * @returns {boolean} `true` if the element is visible; otherwise, `false`.
+     */
+    const checkElementVisibility = (element) => {
```

Comment:
it's a nitpicking, but can you consider renaming this method to isElementVisible?

Reply:
Done